docs/root/version_history/v1.12.0.rst

```diff
@@ -28,13 +28,13 @@ Changes
 * config: added async data access for local and remote data sources.
 * config: changed the default value of :ref:`initial_fetch_timeout <v1.12:envoy_api_field_core.ConfigSource.initial_fetch_timeout>` from 0s to 15s. This is a change in behaviour in the sense that Envoy will move to the next initialization phase, even if the first config is not delivered in 15s. Refer to :ref:`initialization process <v1.12:arch_overview_initialization>` for more details.
 * config: added stat :ref:`init_fetch_timeout <v1.12:config_cluster_manager_cds>`.
-* config: tls_context in Cluster and FilterChain are deprecated in favor of transport socket. See :ref:`deprecated documentation <v1.12:deprecated>` for more information.
+* config: tls_context in Cluster and FilterChain are deprecated in favour of transport socket. See :ref:`deprecated documentation <v1.12:deprecated>` for more information.
```

Comment:
with these us/uk spellings i would suggest jsut leaving them as they are - this one switches to uk spelling and below its doing the reverse